AbletonMCP_Remote_Script/handlers/dispatch.py

```
@@ -0,0 +1,259 @@
+"""
+
+from __future__ import absolute_import, print_function, unicode_literals
+
+from . import tracks, browser, devices, arrangement, audio
+
+
+def _resolve_track(song, track_index, track_type):
+    """Resolve a track object from index and type."""
+    if track_type == "return":
+        if track_index < 0 or track_index >= len(song.return_tracks):
+            raise IndexError("Return track index out of range")
```

Ruff (0.15.1)

[warning] Avoid specifying long messages outside the exception class

(TRY003)